geode-protobuf/src/main/java/org/apache/geode/protocol/protobuf/ProtocolErrorCode.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License), Version 2.0
+ * (the "License")), you may not use this file except in compliance with the License. You may obtain
+ * a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE2.0
+ *
+ * Unless required by applicable law or agreed to in writing), software distributed under the
+ * License is distributed on an "AS IS" BASIS), WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND),
+ * either express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.geode.protocol.protobuf;
+
+public enum ProtocolErrorCode {
+  GENERIC_FAILURE(1000),
```

Comment:
I believe any error code structure decides how many characters are to be used for an error code. 
The fact that we represent them as integers is merely a "simplification" choice over using Alpha numerics.
I don't believe starting the error code at 1000 is a problem